site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -262,6 +278,24 @@ export const WorkspaceScheduleForm: FC<WorkspaceScheduleFormProps> = ({
   )
 }
 
+const ttlShutdownAt = (now: dayjs.Dayjs, workspace: Workspace, tz: string, newTTL: number): string => {
+  if (workspace.latest_build.transition !== "start") {
+    return Language.ttlHelperText
+  }
```

Comment:
I have a `util` for this:

https://github.com/coder/coder/blob/fff59ef6ad4e1d9026b228e8d2164004a5c7ff60/site/src/util/workspace.ts#L246-L250